ci-operator/config/openshift/router/openshift-router-master.yaml

```diff
@@ -140,6 +140,26 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-upgrade-azure
+- always_run: false
+  as: perfscale-aws-ingress-perf
+  optional: true
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "9"
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      LOKI_USE_SERVICEMONITOR: "false"
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: c5.4xlarge
```

Comment:
Can we specify other env variables, e.g. for FIPS, ovn, number of infra nodes, number of control plane nodes, or other?  What does `SET_ENV_BY_PLATFORM: custom` indicate?

Reply:
This is for `infra` node sizing. 

We could consider adding a FIPS job outside of this PR.